bin/run_sql_task.py

```diff
@@ -45,6 +45,11 @@ def main():
                 "db_user": parse_dsn(settings["database_url"].strip())["user"],
                 "region": Regions.get_region(),
                 "h_fdw": parse_dsn(settings["h_fdw_database_url"]),
+                "h_fdw_tables": [
+                    "authorities",
+                    "annotation_group_counts",
+                    "annotation_user_counts",
+                ],
```

Comment:
It makes sense to declare this stuff once, as we don't want these things to drift apart, but it still makes me a bit uncomfortable to have the data mixed in with the config.